Add Redis integration to the project

Malexander307 opened the pull request:
- Added Redis environment variables in `.env.example`
- Modified `docker-compose.yml` to include Redis service
- Updated `go.mod` and `go.sum` to include Redis dependencies
- Created `internal/configs/redis.go` to establish Redis connection
- Implemented `RedisService` in `internal/services/redis_service.go` with basic operations (Ping, Set, Get)

Malexander307 committed:
Add Redis integration to the project

- Added Redis environment variables in `.env.example`
- Modified `docker-compose.yml` to include Redis service
- Updated `go.mod` and `go.sum` to include Redis dependencies
- Created `internal/configs/redis.go` to establish Redis connection
- Implemented `RedisService` in `internal/services/redis_service.go` with basic operations (Ping, Set, Get)

VitaliiFedorchenko commented on docker-compose.yml:
```diff
@@ -13,11 +13,15 @@ services:
       DB_PASSWORD: ${DB_PASSWORD}
       DB_HOST: ${DB_HOST}
       DB_NAME: ${DB_NAME}
+      REDIS_HOST: redis
```
change to ${REDIS_HOST}

VitaliiFedorchenko commented on go.mod:
```diff
@@ -30,6 +32,7 @@ require (
 	github.com/josharian/intern v1.0.0 // indirect
 	github.com/leodido/go-urn v1.4.0 // indirect
 	github.com/mailru/easyjson v0.9.0 // indirect
+	github.com/redis/go-redis/v9 v9.7.0 // indirect
```
Add link for new lib to README

Malexander307 committed:
Add Redis support and clean up dependencies

- Updated `README.md` to include `go-redis` as a dependency.
- Modified `docker-compose.yml` to use `REDIS_HOST` from environment variables.
